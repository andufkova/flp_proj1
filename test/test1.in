S,A,B
a,b,c
S
S->A
A->ab
A->aAB
A->aBb
A->aABbbcc
A->B
B->a
B->B
B->A
B->BA
