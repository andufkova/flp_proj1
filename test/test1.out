S,A,B
a,b,c
S
A->ab
A->aAB
A->aBb
A->aABbbcc
B->a
B->BA
S->ab
S->aAB
S->aBb
S->aABbbcc
S->a
S->BA
A->a
A->BA
B->ab
B->aAB
B->aBb
B->aABbbcc
